FakeServer/GraphQL/GraphQLMiddleware.cs

```diff
@@ -75,5 +91,28 @@ public GraphQLMiddleware(RequestDelegate next, IDataStore datastore, IMessageBus
 
             await context.Response.WriteAsync(json);
         }
+
+        private static async Task<string> ParseQuery(HttpContext context)
+        {
+            var body = string.Empty;
+
+            using (var streamReader = new StreamReader(context.Request.Body))
+            {
+                body = await streamReader.ReadToEndAsync().ConfigureAwait(true);
+            }
+
+            if (context.Request.ContentType == "application/graphql")
```

Comment:
Some tests fail on this check as ContentType is actually `application/graphql; charset=utf-8`